src/main/java/net/sf/jsqlparser/expression/ArrayExpression.java

```diff
@@ -72,11 +74,7 @@ public String toString() {
         if (indexExpression != null) {
             return objExpression.toString() + "[" + indexExpression.toString() + "]";
         } else {
```

Comment:
This is a perfect example for a reject: The Format may be Google style compliant, but it is harder to read how/why the string is composed. The logic is lost.